### ISSUE TITLE
Temperature log export with no logs causes indefinite load spinner animation

### ISSUE THREAD

**Issue by katherine-sussol** [openmsupply/mobile#3747]
## Describe the bug
Attempting to email a temperature log report with no logs results in the app displaying the fancy orange spinner which doesn't really go away on it's own:
![screen](https://user-images.githubusercontent.com/65875762/110865515-9e23aa00-8328-11eb-86b1-b2a1a248646e.png)

Logged is the following message:
```
[Fri Mar 12 2021 11:42:54.399]  WARN     Possible Unhandled Promise Rejection (id: 0):
Error: No temperature logs
emailVaccineReport$@http://localhost:8081/index.bundle?platform=android&dev=true&minify=false:355875:28
...
```

### To reproduce

1. Set up a new sensor
2. Click the icon to download csv logs before any logs have downloaded
3. Enter an email address and comment and click 'download' 
4. Observe Error

### Expected behaviour

Probably just a user-friendly toast or something

### Proposed Solution

Leave if you don't know how to fix/implement. Edit this issue description and explain here if you know the best path of implementing the fix within the codebase.

### Version and device info

- App version: V7.0.0 RC10
- Tablet model: Lenovo
- OS version: Android 8.1

### Additional context

Don't think this is critical as only silly or really impatient users would try to download logs when they know they've just set it up.

**Comment by katherine-sussol** [comment]
Closed with #3936

**Comment by ArpitaSussol** [comment]
Tested in V8.1.0 rc1

## Testing

- Setup a new sensor with a log delay of 5min and try to export temperature logs before logging is started (from vaccines page).
  - [x]  Check that a toast is displayed instead of a continuous spinner animation
